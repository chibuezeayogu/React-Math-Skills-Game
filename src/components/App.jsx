import React from 'react';

const App = () => (
 <div>
   <h1>
     Play Nine
   </h1>
 </div>
);

export default App;
