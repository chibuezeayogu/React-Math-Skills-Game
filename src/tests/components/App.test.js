/* 
global jest 
expect 
*/
import React from 'react';
import { mount } from 'enzyme';
import App from '../../components/App';

describe('App Component', () => {
  it('should render correctly', () => {
    const wrapper = mount(<App />);
    
    expect(wrapper).toMatchSnapshot();
  });
});
