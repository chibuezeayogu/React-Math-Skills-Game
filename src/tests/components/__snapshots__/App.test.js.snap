// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component should render correctly 1`] = `
<App>
  <div>
    <h1>
      Play Nine
    </h1>
  </div>
</App>
`;
